src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(nullable = false, unique = true)
+    private String email;
+    @Column(nullable = false)
+    private String password;
+    @Column(name = "first_name", nullable = false)
```

Comment:
```suggestion
    @Column(nullable = false)
```

Reply:
Avoid:
camelCase - it is difficult to scan quickly.

https://www.sqlstyle.guide/#naming-conventions

Reply:
It's not related to SQL, it's related to Hibernate - automatically generate snake_case column name, so you don't need to specify it.